### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by skayliu**
## Description

Update the Release Lifecycle of `8.6-alpha` to `8.7-alpha`

## Checklist

- [ ] for CI changes:
  - [ ] structural/foundational changes signed off by [CI DRI](https://github.com/cmur2)
  - [ ] [ci.yml](https://github.com/camunda/camunda/blob/main/.github/workflows/ci.yml) modifications comply with ["Unified CI" requirements](https://github.com/camunda/camunda/wiki/CI-&-Automation#workflow-inclusion-criteria)

## Related issues

closes #

### CODE REVIEW

**Review by korthout — APPROVED** [review]
Good catch! Thanks @skayliu 

LGTM 👍

**Comment by korthout** [comment]
Due to [a limitation in our current CI](https://github.com/camunda/camunda/issues/19222), I have to recreate your pull request. I have done so in https://github.com/camunda/camunda/pull/22242.

> Make sure the external contributor understands this that they will be properly credited (Camunda does not want to steal any code, this is just for technical reasons and will be done on behalf of the contributor)

I have reworded the commit, but you remain the author of the commit. I have kept all changes as they were. Thank you again for your contribution @skayliu 🙇

**Comment by skayliu** [comment]
> Due to [a limitation in our current CI](https://github.com/camunda/camunda/issues/19222), I have to recreate your pull request. I have done so in #22242.
> 
> > Make sure the external contributor understands this that they will be properly credited (Camunda does not want to steal any code, this is just for technical reasons and will be done on behalf of the contributor)
> 
> I have reworded the commit, but you remain the author of the commit. I have kept all changes as they were. Thank you again for your contribution @skayliu 🙇

@korthout That's ok, just do it.

File: README.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ Here is a diagram illustrating the release schedule of the five alpha releases p
 2024                                2025
 Oct       Nov          Dec          Jan          Feb          Mar          Apr
 8.6-----------------------------------------------------------------------------
-          8.6-alpha1   8.6-alpha2   8.6-alpha3   8.6-alpha4   8.6-alpha5   8.7--
+          8.7-alpha1   8.7-alpha2   8.7-alpha3   8.7-alpha4   8.7-alpha5   8.7--
```
Comment (tmetzke) [inline README.md:58]: I guess 8.7 is due in January, isn't it?
```suggestion
          8.7-alpha1   8.7-alpha2   8.7   8.8-alpha1   8.8-alpha2   8.8--
```

Comment (korthout) [reply README.md:58]: 👍 Let's tackle that [separately](https://camunda.slack.com/archives/C05DH1F5TAR/p1726068183693239).

**Comment by korthout** [comment]
Merged with #22242

**Pull request closed without merging**